src/components/split/splitInputs.js

```diff
@@ -0,0 +1,14 @@
+import Calculator from '../calculation/calculator.js';
+
+export default function splitInputs(inputs) {
+    let splitedInputs = [];
+    if (inputs.includes(':') === true) {
+        splitedInputs = inputs.split(':');
+    } else if (inputs.includes(',') === true) {
+        splitedInputs = inputs.split(',');
+    }
+
+    const splitedNum = splitedInputs.map((num) => Number(num));
```

Comment:
정~~말 별거 아니긴 한데 혹시 귀찮으시다면 map(Number)로 훨씬 간단하게 쓸 수 있습니다! 👍
```suggestion
    const splitedNum = splitedInputs.map(Number);
```